src/pkg/bundle/common.go

```diff
@@ -327,10 +327,22 @@ func validateBundleVars(packages []types.Package) error {
 		// ensure imports have a matching export
 		if pkg.Imports != nil {
 			for _, v := range pkg.Imports {
-				if _, ok := exports[v.Name]; ok && v.Package == exports[v.Name] {
-					continue
+				// check import names if they are set, otherwise importing all the variables for the given package
```

Comment:
As a newcomer to the project, I've been familiarizing myself with the codebase and noticed a section in the import package validation logic that caught my attention. Specifically, it pertains to the logic block that validates import packages without specified variable names against exported packages. My understanding is that the current implementation might prematurely flag an import package as invalid if the first exported package name checked does not match the import package name, without considering the rest of the exported packages.

### Suggested Improvement:
I believe there might be an opportunity to enhance the validation process's accuracy by ensuring all exported package names are checked against the import package name before determining a mismatch. However, I acknowledge that I might not have full context or might be overlooking other considerations, so I'd appreciate any feedback or corrections.

### Proposed Changes:
```go
// check import names if they are set, otherwise importing all the variables for the given package
if v.Name != "" {
    if _, ok := exports[v.Name]; ok && v.Package == exports[v.Name] {
        continue
    }
    return fmt.Errorf("import var %s does not have a matching export", v.Name)
    // Check that the import package is valid
} else {
    // Initialize a variable to track if the package is found
    var packageFound bool
    for _, pkgName := range exports {
        if pkgName == v.Package {
            packageFound = true
            break
        }
    }
    // If the package is not found in exports, return an error
    if !packageFound {
        return fmt.Errorf("import package %s does not match any exporting package", v.Package)
    }
}
```

### Rationale:
Implementing this change could potentially make the validation logic more robust by ensuring that all exported packages are considered before concluding that an import package does not match. This could help in avoiding false negatives in the validation process and improve the overall reliability of the package validation mechanism.

### Impact:
This proposed change is intended to refine the validation logic without adversely affecting existing functionalities. It aims to enhance the accuracy of the package validation process.

As I'm new to this project, I'm aware that there might be aspects I'm not fully aware of or additional context that I might be missing. Therefore, I'm very open to feedback, corrections, or any insights that could help me understand this better. Thank you for considering my suggestion, and I look forward to any discussions this might prompt.

Reply:
Good catch! will be updated in next commit. Thanks @naveensrinivasan !

Reply:
So we actually got rid of the above code. We decided not to support importing variables by just specifying the package name they are being exported from. We are going to make all exported variables available by default, and in the case of name collisions, you can import variables from specific packages like before.